File: 3555-final-array-state-after-k-multiplication-operations-i/3555-final-array-state-after-k-multiplication-operations-i.java
class Solution {
    public int[] getFinalState(int[] nums, int k, int mul) {
        for(int i=0;i<k;i++)
        {
            int min=nums[0],ind=0;
            for(int j=1;j<nums.length;j++)
            {
                if(nums[j]<min)
                {
                    min=nums[j];
                    ind=j;
                }
            }
            nums[ind]=min*mul;

        }
    return nums;   
    }
}

File: 3555-final-array-state-after-k-multiplication-operations-i/3555-final-array-state-after-k-multiplication-operations-i.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, mul: int) -> List[int]:
        for i in range(k):
            min=nums[0]
            ind=0
            for j in range(1,len(nums)):
                if nums[j]<min:
                    min=nums[j]
                    ind=j
            nums[ind]*=mul
        return nums

        